src/components/common/dashboard/Navbar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Link, Button, Flex, Heading, Icon, Image, VStack } from '@chakra-ui/react';
+import { FiUsers } from 'react-icons/fi';
+import { IoHomeOutline, IoPodiumOutline } from 'react-icons/io5';
+import { Link as RouterLink, NavLink } from 'react-router-dom';
+import Logo from '../../../assets/logoWhite.png';
+
+const NavItem = props => (
+  <Link as={NavLink} to={props.to} _activeLink={{ color: 'white' }}>
+    <Heading size="h4" as="span">
+      <Flex>
+        <Icon as={props.icon} mr={2} />
+        {props.children}
+      </Flex>
+    </Heading>
+  </Link>
+);
+
+const Navbar = props => (
+  <Flex
+    as="nav"
+    bg="background.secondary"
+    direction="column"
+    h="100%"
+    px={12}
+    py={20}
+    justify="space-between"
+    overflowX="hidden"
+    textTransform="capitalize"
+    role="navigation"
+  >
+    <VStack color="gray.400" spacing={10} align="flex-start">
+      <Image src={Logo} alt="Raising The Roof Logo" w="100%" />
+      <NavItem to="/home" icon={IoHomeOutline}>
```

Comment:
I think this is more of a PM/design choice but when I click to the right of the actual labels, do we want that to also take us to the link?

Reply:
// will ask design this

Reply:
https://uwblueprint.slack.com/archives/C01JT2AE91R/p1614567766004800?thread_ts=1614563841.001900&cid=C01JT2AE91R